### DISCUSSION TITLE
How can I build the project without js minimization?

### DISCUSSION THREAD

**Discussion by muberraduman** [GrapesJS/grapesjs#3949] [Q&A]
I am getting different errors when I drag things into the editor and the error message points to "grapes.min.js:2"

Is there a way to build the project without minimization, so I can get a better idea about the problem?

ie: Cannot read properties of undefined (reading 'el') at Object.getContainer (grapes.min.js:2)

**Comment by ronaldohoch** [comment]
You have the same error if you use the file from unpkg? 
`<script src="https://unpkg.com/grapesjs"></script>`

But, you can clone this repo and run it.
